build.gradle

```diff
@@ -585,7 +585,6 @@ jlink {
     // but it was removed in the newer releases.
     // The pom.xml associated with such a non-modular artifact does not mention that the artifact depends on the removed code
     // (because the artifact was published when this code was still available in the JDK).
-    forceMerge "javafx"
```

Comment:
this will not work this is intended for our merged module.

Reply:
you could try to add gemsfx to the merged module declaration

Reply:
Based on the comments above, I was thinking, it could help.

    Two versions of module javafx.graphics found in /Users/runner/work/***/***/build/jlinkbase/jlinkjars (javafx-graphics-20-mac.jar and javafx.graphics.jar)

Reply:
This sounds like the zulu already ships with javafx and now it cannot decide which one to use because we get javafx over the plugin

Reply:
/home/runner/.gradle/caches/modules-2/files-2.1/org.openjfx/javafx-base/20/d82ec648e48647e10125ff0e3dce70348a80ac18/javafx-base-20-linux.jar

Reply:
This is likely the main culprit: 
https://github.com/JabRef/jabref/blob/b140d8a81ba63c439e4502c70c286fcaa590ca2d/build.gradle#L104-L107

Reply:
(I document more for myself than for others)

> This is likely the main culprit:

No. 

This points to JavaFX 22, not 22. The file included is ` /home/runner/.gradle/caches/modules-2/files-2.1/org.openjfx/javafx-base/20/d82ec648e48647e10125ff0e3dce70348a80ac18/javafx-base-20-linux.jar` and thus is 20 (twenty) not 22 (twenty-two)

I replace `version` by `sdk` in the `yaml` workflow: https://github.com/JabRef/jabref/blob/f6a7217983d82ea236d6ab3b18643643682e81bb/.github/workflows/deployment-jdk-ea.yml#L166

Reply:
I seee,  but they are still duplicated on the classpath

Reply:
Only with the `-20` or do you see other duplicates?

Reply:
Maybe https://github.com/beryx/badass-jlink-plugin/issues/61#issuecomment-504640018 helps. Currently checking.